adapter/service/outlvlswitch/routing_test.go

```diff
@@ -0,0 +1,263 @@
+package outlvlswitch_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/service/outlvlswitch"
+	"github.com/futurehomeno/cliffhanger/router"
+	"github.com/futurehomeno/cliffhanger/task"
+	adapterhelper "github.com/futurehomeno/cliffhanger/test/helper/adapter"
+	mockedadapter "github.com/futurehomeno/cliffhanger/test/mocks/adapter"
+	mockedoutlvlswitch "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/outlvlswitch"
+	"github.com/futurehomeno/cliffhanger/test/suite"
+)
+
+func TestRouteService(t *testing.T) { //nolint:paralleltest
+	s := &suite.Suite{
+		Cases: []*suite.Case{
+			{
+				Name:     "Should succeed processing commands",
+				TearDown: adapterhelper.TearDownAdapter("../../testdata/adapter/test_adapter"),
+				Setup: routeService(
+					mockedoutlvlswitch.NewMockedOutSwitchLvl(
+						mockedoutlvlswitch.NewController(t).
+							MockLevelSwitchLevelReport(2, nil, false).
```

Comment:
Please make level report more diversified, so every action against the controller ends up with an actual level transition.